### ISSUE TITLE
code for training

### ISSUE THREAD

**Issue by emseel** [iro-cp/FCRN-DepthPrediction#24]
I cannot find the code that is used to train your model, will you be making it available soon please? I would like to experiment with different data sets. :)

**Comment by codepujan** [comment]
@emseel  I agree .

**Comment by Lvhhhh** [comment]
i agree~

**Comment by yjl9122** [comment]
I rewrite train code. But i can't achieve 0.127, is there anyone can reach 0.127?

**Comment by Pratapabhay** [comment]
@yjl9122 Hi, Can you share training code?

**Comment by yjl9122** [comment]
I train the network  without anything special. So you can write one. I think it not easy to train. I think it has many tricks

**Comment by Pratapabhay** [comment]
Hi,
Thanks for the reply. Thing is I am not very experienced with working on
tensor flow, but I need to train the model for my project. It would be
really helpful to me if you share your training code, if its okay with you.

Thanks

On Aug 25, 2017 9:40 PM, "yjl9122" <notifications@github.com> wrote:

I train the network without anything special. So you can write one. I think
it not easy to train. I think it has many tricks

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub
<https://github.com/iro-cp/FCRN-DepthPrediction/issues/24#issuecomment-324966031>,
or mute the thread

**Comment by fangchangma** [comment]
> I train the network without anything special. So you can write one. I think it not easy to train. I think it has many tricks

I agree. I set up the same model with "DeConv" layers and train with L2 loss function (i.e., the simplest architecture in the paper). In the paper it says 0.152 (rel error), but the best I can get so far is around 0.18 (so roughly 20% higher than the paper).  

It will be really helpful if the training script is released.

**Comment by yjl9122** [comment]
oh, sorry, I just receive your message. If you are not very experienced with tensorflow, you can use deeplab training code https://github.com/DrSleep/tensorflow-deeplab-resnet. This is a very good!

**Comment by JackHenry1992** [comment]
Hi, @yjl9122 , I also looking for a trainning code as reference, can you share your trainning code ?

**Comment by skylook** [comment]
Me too, need a training code to test on our own database

**Comment by retkowski** [comment]
@iro-cp : Would you mind to share the training code? Thanks :)

**Comment by somsol** [comment]
Hi, would appreciate if you could share the matconvnet training code. Thanks in advance!
